packages/cli/src/workflows/workflow.service.ts

```diff
@@ -69,6 +69,7 @@ export class WorkflowService {
 		options?: ListQuery.Options,
 		includeScopes?: boolean,
 		includeFolders?: boolean,
+		onlySharedWithMe?: boolean,
```

Comment:
I suppose these are never actually undefined as we always call this function with these set but 🤷